app/controllers/index.js

```diff
@@ -1,8 +1,7 @@
 const User = require("./user.controller");
-const model = require("../../database/models");
+const service = require("../../database/acid/services");
 const {hashPass, comparePass} = require("../helpers/auth.bcrypt");
```

Comment:
Take advantage of using ES2015 feature. Use `import User from './user.controller'`. Apply this in all the files.